src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/QueryIT.java

```diff
@@ -99,6 +99,59 @@ public void multipleFromTest() throws IOException {
         Assert.assertEquals(14, getTotalHits(response));
     }
 
+    @Test
+    public void selectAllAndSpecificFieldTest() throws IOException {
+        String[] arr = new String[] {"account_number", "firstname", "address", "birthdate", "gender", "city", "lastname", "balance", "employer", "state", "age", "email", "male"};
+        String fieldName = "age";
+        Set<String> expectedSource = new HashSet<>(Arrays.asList(arr));
+        JSONObject response = executeQuery(String.format(Locale.ROOT,
+                "SELECT *, %s as t " +
```

Comment:
So what is the expected behavior : should return all the fields and "t" or just all the fields?

Reply:
Should return all the fields which naturally includes t

Reply:
I think this should return all the fields with original names, plus the extra "age" field but named "t". Please confirm with a Sql or MySql implementation. We still have many issues in our plugin, so please do not completely rely on it when determining the correct behavior.

Reply:
Alright, so I tested with different online editors (claiming to be either Sql or MySql compliant), and in all cases there is a column with the alias. However, in some cases the original column was there, in some cases not, e.g.

```sql
create table Test(id integer, title varchar(100));
insert into Test(id, title) values(1, "Hello");
insert into Test(id, title) values(2, "World");
insert into Test(id, title) values(3, "blah");

select *, title as word from Test;
```

In some cases this results in:
```
id  |  title |  word
...     ...     ...
```

and in others
```
id  |  word
...    ...
```

I am more inclined to support the first option -> all fields + alias, to be consistent with queries like `SELECT someField, someField AS otherField FROM tableX` which always returns 2 columns.

Reply:
Verified and you are right- will be more thorough testing in the future. Currently opendistro does not rename the column correctly, as it still shows up as `address`. Should I open up another issue for the column alias?

Reply:
Had lots of trouble fixing column alias bug- Chen observed that aliases are supported with JDBC format, including the original issue. Left in original change that will return all fields in query with * and selected field